multigeiger/speaker.cpp

```diff
@@ -58,17 +58,17 @@ void IRAM_ATTR isr_audio() {
       playing_tick = true;
     }
   }
-  if (isr_sequence) {
-    volatile int *p = isr_sequence;
+  volatile int *p = isr_sequence;
+  if (p) {
     frequency_mHz = *p++;
     volume = *p++;
     led = *p++;
     duration_ms = *p++;
-    isr_sequence = p;
+    isr_sequence = duration_ms == 0 ? NULL : p;
```

Comment:
i think this is not needed, see line 108.

Reply:
removed, chose a different solution by relying on playing_tick / playing audio bools.